public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -164,30 +170,14 @@ export class PanelEditorUnconnected extends PureComponent<Props> {
     ) {
       try {
         await saveAndRefreshLibraryPanel(this.props.panel, this.props.dashboard.meta.folderId!);
-        this.props.updateSourcePanel(this.props.panel);
```

Comment:
I removed these two calls to updateSourcePanel because I could not see these having any effect, in all the "leave panel edit" flows we never compared changes to updating sourcePanel had no effect that I could see

Reply:
Maybe @kaydelaney has some insight on this?

Reply:
No insight I'm afraid. :(

Reply:
I can't remember either, so we're all good :)